pkg/agent/facts/factsopenshift.go

```diff
@@ -48,6 +54,24 @@ func (col factCollector) fetchOpenshiftVersion(ctx context.Context) (*SemanticVe
 	return processOpenshiftVersion(version)
 }
 
+func (col FactCollector) fetchOpenshiftOAuthRoute(ctx context.Context) (string, error) {
+	body, err := col.Client.RESTClient().Get().AbsPath("/apis/route.openshift.io/v1/namespaces/openshift-authentication/routes/oauth-openshift").Do(ctx).Raw()
```

Comment:
should we make the route namespace & name configurable? or are we fine with patching the code if this ever changes?